Nos/Views/DiscoverView.swift

```diff
@@ -109,7 +117,7 @@ struct DiscoverView: View {
             )
             .autocorrectionDisabled()
             .onSubmit(of: .search) {
-                submitSearch()
+                searchController.submitSearch()
```

Comment:
@mplorentz I think we discussed removing this `onSubmit` but we'll still need to call `submitSearch()` somewhere, and again I'm not familiar enough with the code to know where.